LDAR_Sim/src/weather/weather_lookup.py

```diff
@@ -29,46 +29,51 @@ def __init__(self, virtual_world, input_directory):
         """
 
         self.virtual_world = virtual_world
-
+        self.input_directory = input_directory
+
+        # Initialize attributes to store weather data
```

Comment:
Question: Why do all these need to get stored?

Reply:
I'm not sure if this is the best way to do this, and I'm open to feedback. Weather needs to be serializable.
The reason the virtual world and input directory are stored is such that when the weather object is reconstructed, it would re-load the data, essentially re-read in the inputs... This does mean that we are doing more I/O than needed and possibly will slow down the simulation load time.

Reply:
Sorry, let me be more specific, I didn't see these getting used anywhere beyond to calculate other results. So why are we storing them instead of just calculating the results and storing that?

Reply:
Changed.